Comment out config builder section on `upboard` (#1730)
viamrobotics/docs: docs/components/board/upboard.md

---

Configure an `upboard` board to integrate an Intel-based board like the [UP4000](https://github.com/up-board/up-community/wiki/Pinout_UP4000) into your robot.

> Xomplete the following set-up requirements, then move on to configuring your board in the Viam app](https://app.viam.com):

Author: ```suggestion
Complete the following set-up requirements, then move on to configuring your board in the Viam app](https://app.viam.com):
```

> Complete the following set-up requirements, then move on to configuring your board in the Viam app](https://app.viam.com):

Author: @JessamyT Feedback

> Complete the following set-up requirements, then move on to configuring your board in [the Viam app](https://app.viam.com):

Reviewer: ```suggestion

Complete the following setup requirements, then move on to configuring your board in [the Viam app](https://app.viam.com):
```
I think we've been doing setup as one word when it's a noun like this. LGTM thx for adding.

Follow [these instructions](https://github.com/up-division/pinctrl-upboard) to do so.
This driver stabilizes the [GPIO pin mapping definition](https://github.com/up-board/up-community/wiki/Pinout) on the board to make it identical to that of a [Raspberry Pi](/components/board/pi/).

<!-- {{< tabs name="Configure an upboard Board" >}}

> {{% tab name="Config Builder" %}}

Author: Commented out as it's not actually visible in the app, so you can't use the config builder for it. At least it isn't for me right now.

Edit and fill in the attributes as applicable.

{{% /tab %}}
{{% tab name="JSON Template" %}} -->

Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
Click on the **Components** subtab and navigate to the **Create component** menu.

> Select **Raw JSON** mode and copy and paste the following:

Author: @andf-viam Ahhh, thank you for catching json here!! I think something really weird happens when commenting out tabs with hugo. It renders now for me in the GH preview, does it render for you?

Reviewer 2: It does! And I think you're right about this being a very-hugo very-limitation. If not you, here, this would have been me, elsewhere!! Thanks for fixing!

> Select **Raw JSON** mode and copy and paste the following: 

Author: ```suggestion
Select **Raw JSON** mode and copy and paste the following:
```
